bitnami/kibana/templates/serviceaccount.yaml

```diff
@@ -0,0 +1,6 @@
+{{- if.Values.serviceAccount.create }}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name:  {{ required .Values.serviceAccount.name }}
+{{- end }}
```

Comment:
Could you please base on the following service account in order to standardize it please? https://github.com/bitnami/charts/blob/master/bitnami/redis/templates/serviceaccount.yaml